lib/setup/setupSetup.js

```diff
@@ -264,21 +281,35 @@ function Setup(options) {
                                 sp = 9000;
                             } else if (stype === 'redis') {
                                 sp = 6379;
+                                if (shost.includes(',')) {
+                                    shost = shost.split(',');
+                                    shost.forEach((host, idx) => {
+                                        shost[idx] = host.trim();
+                                    });
```

Comment:
`shost = shost.split(',').map(host => host.trim());`

Reply:
used, I still need tzo think into these style of code :-)